optimum/habana/transformers/training_args.py

```diff
@@ -218,6 +273,9 @@ def __post_init__(self):
         if env_local_rank != -1 and env_local_rank != self.local_rank:
             self.local_rank = env_local_rank
 
+        if self.local_rank != -1 and self.use_hpu_graphs_for_training and self.distribution_strategy != "fastddp":
+            raise ValueError("--use_hpu_graphs_for_training may only be used with --distribution_strategy fastddp")
```

Comment:
Do you think HPU graphs bring any speedup for training on a single device?

Reply:
Oh yes.